modules/coreos-layering-configuring.adoc

```diff
@@ -147,8 +147,9 @@ $ oc get mcp
 [source,terminal]
 ----
 NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
-master   rendered-master-6faecdfa1b25c114a58cf178fbaa45e2   True      False      False      3              3                   3                     0                      39m
-worker   rendered-worker-6b000dbc31aaee63c6a2d56d04cd4c1b   False     True       False      3              0                   0                     0                      39m <1>
+master   rendered-master-15961f1da260f7be141006404d17d39b   True      False      False      3              3                   3                     0                      39m
+worker   rendered-worker-5aff604cb1381a4fe07feaf1595a797e   False     True       False      3              0                   0                     0                      39m <1>
```

Comment:
```suggestion
worker   5bdb57489b720096ef912f738b46330a8f577803   False     True       False      3              0                   0                     0                      39m <1>
```

Reply:
this is the tag for the `CREATEDBYCONTROLLER`, it goes here?

Reply:
It does not. That should be the rendered MC.